### PR TITLE
feat: Automatically label issues

### PR DESCRIPTION
**Pull request by teolemon**
### Checklist
- Automatically label issues

### CODE REVIEW

File: .github/issue-labeler.yml [hunk 1]
```diff
@@ -0,0 +1,1019 @@
...
+no-pr-activity:
+    - '(no-pr-activity)'
+
+nodejs:
+    - '(nodejs)'
+
+non tech:
+    - '(non tech)'
+
+not done
```
Comment (stephanegigandet) [inline .github/issue-labeler.yml:581]: typo?

**Review by stephanegigandet — CHANGES_REQUESTED** [review]
issue-labeler can only be used for a very small and specific set of labels that are expected in an issue template.

It's probably not a good match for this repo that has lots of labels.

Even on other repos with fewer labels, please try it with one label at a time, and uncomment the not-before option, so that we can check that it does not break old issues.

Specifically, merging this PR as-is would remove almost all existing labels on almost all issues.

File: .github/issue-labeler.yml (hunk 1 above)
Comment (teolemon) [reply .github/issue-labeler.yml]: fixed

**Comment by teolemon** [comment]
@stephanegigandet I've disabled processing of issues older than today, and enabled a revision mechanism

**Comment by stephanegigandet** [comment]
@teolemon : this PR will remove most labels entered manually, unless their body also match the exact regexps (including case) listed.

**Comment by stephanegigandet** [comment]
@teolemon example : I changed the title of this issue https://github.com/openfoodfacts/smooth-app/issues/1166 , and the action removed the ios label, because it is not in the body.

**Comment by stephanegigandet** [comment]
Closing as issue-labeler does not work as wanted.

**Pull request closed without merging**